site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -13,25 +13,41 @@ namespace as it - usually referred to in Kubernetes terms as a "sidecar".
 
 Therefore, when developing locally, we also need a process for the SDK to connect to!
 
-To do this, we can run the same binary that runs inside Agones, but pass in a flag
+To do this, we can run the same binary (the SDK Server) that runs inside Agones, but pass in a flag
 to run it in "local mode". Local mode means that the sidecar binary
 will not try to connect to anything, and will just send log messages to stdout and persist local state in memory so
 that you can see exactly what the SDK in your game server is doing, and can
 confirm everything works.
 
-To do this you will need to download {{% ghrelease file_prefix="agonessdk-server" %}}, and unzip it.
+For local development with integration into your cluster, the SDK Server can also be run in "out of cluster mode", discussed more [below](#running-locally-using-out-of-cluster-mode).
+
+## Running the SDK Server
+
+To run the SDK Server, you will need a copy of the binary.
+This can either be done by downloading a prebuilt binary or running from source code.
+
+### Getting the prebuilt binary
+
+To run the prebuilt binary, for the latest release, you will need to download {{% ghrelease file_prefix="agonessdk-server" %}}, and unzip it.
 You will find the executables for the SDK server, for each type of operating system.
+`sdk-server.darwin.amd64` and `sdk-server.darwin.arm64` are for <u>MacOS</u>, `sdk-server.linux.amd64` and `sdk-server.linux.arm64` are for <u>Linux</u>, and `sdk-server.windows.amd64.exe` is for <u>Windows</u>.
 
-macOS
-* sdk-server.darwin.amd64
-* sdk-server.darwin.arm64
+### Running from source code
 
-Linux
-* sdk-server.linux.amd64
-* sdk-server.linux.arm64
+Instead of downloading and running executable binaries from the internet, you can simply run from source code.
+First clone the [Agones GitHub repo](https://github.com/googleforgames/agones).
+You can switch to a specific release's branch/tag or run from main.
+Running from source code will require having golang installed, which can be done by following the instructions [here](https://go.dev/doc/install).
 
-Windows
-* sdk-server.windows.amd64.exe
+With golang installed and the Agones repository cloned, the SDK Server can easily be run with the following command (from the agones clone directory):
+```bash
+go run cmd/sdk-server/main.go
+```
+Note: This command does not contain any of the necessary command line flags used in the following sections.
+It simply serves as an example of how to run from source code instead of running the prebuilt binary.
+Passing commandline flags (e.g. `--local`) will work in the same way.
```

Comment:
```suggestion
With Go installed and the Agones repository cloned, the SDK Server can be run with the following command (from the Agones clone directory):
```bash
go run cmd/sdk-server/main.go --local
/```

Commandline flags (e.g. `--local`) are exactly the same as command line flags when utilising a pre-built binary.
```

Tech writer nit: Some things aren't "easily" or "simple" for some people, so better to remove the adjective.

Reply:
Cleaned up.